lib/assessment.js

```diff
@@ -39,6 +55,14 @@ var select = function select(array, callback, thisArg) {
 // Write a function that returns the first element
 // for which invoking the callback returns a truthy value
 var find = function find(array, callback, thisArg) {
+  var result = true;
+  array.forEach(function(currentValue, index, array) {
+
+  if(result = callback(currentValue, index, array)){
```

Comment:
Same feedback as above applies here. The `result` variable is unnecessary in this function, as well.